user_guide/context.rst

```diff
@@ -263,62 +312,85 @@ As a matter of fact, Behat gives you ability to do just that. You can
 specify arguments required to instantiate your context classes through
 same ``contexts`` setting inside your ``behat.yml``:
 
-.. code-block:: yaml
-
-    # behat.yml
-
-    default:
-        suites:
-            default:
-                contexts:
-                    - MyAwesomeContext:
-                        - http://localhost:8080
-                        - /var/tmp
-
-.. note::
+.. code-block:: php
 
-    Note an indentation for parameters. It is significant:
+    <?php
+    // behat.php
 
-    .. code-block:: yaml
+    use App\Tests\Behat\Context\MyAwesomeContext;
+    use Behat\Config\Config;
+    use Behat\Config\Profile;
+    use Behat\Config\Suite;
 
-        contexts:
-            - MyAwesomeContext:
-                - http://localhost:8080
-                - /var/tmp
+    $profile = new Profile('default')
+        ->withSuite(
+            new Suite('default')
+                ->addContext(MyAwesomeContext::class, [
+                    'http://localhost:8080',
+                    '/var/tmp',
+                ])
+        )
+    ;
 
-    Aligned four spaces from the context class itself.
+    return new Config()
+        ->withProfile($profile)
+    ;
 
 Arguments would be passed to the ``MyAwesomeContext`` constructor in
 the order they were specified here. If you are not happy with the idea
 of keeping an order of arguments in your head, you can use argument
 names instead:
 
-.. code-block:: yaml
+.. code-block:: php
+
+    <?php
+    // behat.php
+
+    use App\Tests\Behat\Context\MyAwesomeContext;
```

Comment:
Especially in this example, as the example context class itself is not namespaced and says it is in `features/bootstrap/MyAwesomeContext.php`:

```suggestion
    use MyAwesomeContext;
```